pkg/logs/input/container/docker.go

```diff
@@ -47,18 +47,20 @@ type DockerTailer struct {
 
 	sleepDuration time.Duration
 	shouldStop    bool
+	isFlushed     chan struct{}
```

Comment:
I think a common go pattern is to call this chan `done` (also true in auditor)

Reply:
I agree but here we don't push a done `event`, we are waiting for a `flush` event